src/Transaction.jsx

```diff
@@ -7,6 +7,30 @@ const TEST_CATEGORY_ID = 'test-category-id';
 const TEST_PAYEE_ID = 'test-payee-id';
 
 export default class Transaction {
+  static fromFirebaseData(key, data) {
+    let inflow = 0.0;
+    let outflow = 0.0;
+    if (data.amountMinor < 0) {
+      outflow = Math.abs(data.amountMinor) / 100;
```

Comment:
Yeah, this is on a separate branch from 5119bbd1b62f4beaf3df5045c9174eb1a8255116. I propose to submit that other change first and then merge it into this branch?